app/helpers/reactjs_helper.rb

```diff
@@ -1,8 +1,25 @@
 module ReactjsHelper
+  def react_component(name, opts = { data: [], flatten_data: false })
```

Comment:
```suggestion
  def react_component(name, data: [], flatten_data: false)
```
it will allow us to use `data` and `flatten_data` right away without change on caller side :)